packages/babel-plugin-proposal-class-properties/test/fixtures/private/tagged-template-static/options.json

```
@@ -0,0 +1,13 @@
+{
+    "presets": [
+      [
+        "@babel/preset-env",
+        {
+          "shippedProposals": true,
+          "targets": {
+            "chrome": "75"
+          }
+        }
+      ]
+    ]
+  }
```

Could you try this instead? Ideally we should only compile private fields and methods, but for some reason it's also compiling classes

```suggestion
{
  "plugins": [
    "proposal-class-properties",
    "proposal-private-methods",
  ]
}
```